var/spack/repos/builtin/packages/openfoam/package.py

```diff
@@ -405,10 +405,7 @@ def setup_minimal_environment(self, env):
     def setup_build_environment(self, env):
         """Sets the build environment (prior to unpacking the sources).
         """
-        # Avoid the exception that occurs at runtime
-        # when building with the Fujitsu compiler.
-        if self.spec.satisfies('%fj'):
-            env.set('FOAM_SIGFPE', 'false')
+        pass
```

Comment:
Can we remove this function completely?

Reply:
maybe, but might still need to revive some hooks there in the future.
If you want to remove it, go ahead but my computer is shutdown for a while.